tests/cases.toml

```diff
@@ -86,4 +86,11 @@
   name = "component diamond"
   cps = "diamond"
   args = ["--cflags-only-I"]
-  expected = "-I/something -I/opt/include"
\ No newline at end of file
+  expected = "-I/something -I/opt/include"
+
+[[case]]
+  name = "Requires version, but version not set"
+  cps = "needs-version.cps"
+  args = ["--modversion"]
+  expected = "Could not find a CPS file for needs-version.cps"
```

Comment:
At a minimum, we should get a troubleshooting page for what to do about "missing" CPS files, but in this case, we can actually detect a CPS file (or not), so that would be helpful to explain to the user.

I could see this actually influencing the designs of our algorithms, by the way. We may need invalid CPS info in certain kinds of graphs just so we can report accurately to users what information was missing, incorrect, or unhelpful.

Reply:
Yeah, I've been treating good error messages as a down the road issue to sort through once we have a litlte more of the core logic and the spec ironed out. We could, fairly easily, collect errors at the levels they're currently be discarded and format them better. when passing them down.